share/sysbuild/cmake/modules/sysbuild_kconfig.cmake

```diff
@@ -73,6 +73,13 @@ if(NOT DEFINED SB_CONF_FILE)
   set(SB_CONF_FILE ${CMAKE_CURRENT_BINARY_DIR}/empty.conf)
 endif()
 
+if(NOT EXISTS "${APPLICATION_CONFIG_DIR}")
+  message(FATAL_ERROR "${APPLICATION_CONFIG_DIR} (APPLICATION_CONFIG_DIR) doesn't exists. "
+          "Unable to lookup sysbuild.conf or other related sysbuild configuration files. "
+	  "Please ensure APPLICATION_CONFIG_DIR points to a valid location."
```

Comment:
*SB_APPLICATION_CONFIG_DIR

Reply:
the reason I left out `SB_` is because `SB_` denotes sysbuild scope, but the variable used is in fact `APPLICATION_CONFIG_DIR`.

Just like an image will fail a build with an invalid variable like:
```
ERROR: setting FOO is bad.
```

and not:
```
ERROR: setting <image>_FOO is bad.
```

Remember, sysbuild also honors APPLICATION_CONFIG_DIR, just like any image will use a user specified APPLICATION_CONFIG_DIR, unless a scope specific value is given as `<image>_APPLICATION_CONFIG_DIR`.